conda_build.recipe/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: numpy
+  version: 1.11.0{{ '+' + environ.get('GIT_DESCRIBE_NUMBER', '0') + '.' + environ.get('GIT_DESCRIBE_HASH', '0') if environ.get('GIT_DESCRIBE_NUMBER', '0') != '0' else "" }}
```

Comment:
So, the version is now mostly fixed, but now it has two `+` in some cases. May need to rework that, but for now it is ok as a POC.